_data/projects/l4v.yml

```diff
@@ -6,6 +6,10 @@ display_name: L4.verified
 description: seL4 specification and proofs
 project_order: 2
 
+repositories:
+  - org: seL4
+    repo: l4v
```

Comment:
It might be worthwhile checking how long it takes to clone l4v for this.  The git clone makerule could be further optimized from what it is currently if L4V takes a while to clone `	git clone --depth=1 https://github.com/$(@:_repos/%=%) $@`

Reply:
nevermind.  it's pretty quick already (< 10secs)

Reply:
Cool, yes, it was fairly quick locally.